### DISCUSSION TITLE
How to use the "Syncfusion Asp.NetCore" Tag helper library to generate chart with this library ?

### DISCUSSION THREAD

**Discussion by sathyarajp** [adoconnection/RazorEngineCore#82] [Q&A]
I have to implement Syncfusion Asp.NetCore chart functionality with this custom razor engine. So please help me to add "Syncfusion Asp.NetCore" Tag helpers with this library to render the chart.

**Comment by kftrans** [comment]
Tag helpers aren't supported yet ... but you could track the issue (and add your voice) at https://github.com/adoconnection/RazorEngineCore/issues/81 ...
